### PR TITLE
Per: fix some typos

### PR DESCRIPTION
**Pull request by UnePierre**
- [x] Fix argument name in comment in example of Per.7
- [x] Use the grammar checker of languagetool.org on chapter "Per" and improve it a little, mainly adding commas.

### CODE REVIEW

File: CppCoreGuidelines.md [hunk 1]
```diff
@@ -14456,7 +14456,7 @@ It simply has nothing to do with concurrency.
         if (int n = free_slots--) return &pool[n];
     }
 
-Here we have a problem:
+Here, we have a problem:
```
Comment (cubbimew) [inline CppCoreGuidelines.md:14459]: I don't think commas after these "here"s are warranted. These aren't discourse markers; the text isn't saying "Here, would you look at that? We have a problem!" It's saying "Here, on that line above, we have a problem."

Though I suppose it's not entirely wrong either way https://english.stackexchange.com/questions/321371

Comment (UnePierre) [inline CppCoreGuidelines.md:14459]:
```suggestion
Here we have a problem:
```

File: CppCoreGuidelines.md [hunk 2]
```diff
@@ -14072,17 +14072,17 @@ Consider a popular technique for providing a handle for storing small objects in
     }
 
 Assume that `Scoped` and `On_heap` provide compatible user interfaces.
-Here we compute the optimal type to use at compile time.
+Here, we compute the optimal type to use at compile time.
```
Comment (UnePierre) [inline CppCoreGuidelines.md:14075]:
```suggestion
Here we compute the optimal type to use at compile time.
```

File: CppCoreGuidelines.md [hunk 3]
```diff
@@ -13436,7 +13436,7 @@ This is even more true for mixed signed and unsigned arithmetic.
         cout << subtract(us, s + 2) << '\n';  // 4294967294
     }
 
-Here we have been very explicit about what's happening,
+Here, we have been very explicit about what's happening,
```
Comment (UnePierre) [inline CppCoreGuidelines.md:13439]:
```suggestion
Here we have been very explicit about what's happening,
```

File: CppCoreGuidelines.md [hunk 4]
```diff
@@ -864,7 +864,7 @@ Avoid errors leading to (possibly unrecognized) wrong results.
         // ...
     }
 
-Here we made a small error in `use1` that will lead to corrupted data or a crash.
+Here, we made a small error in `use1` that will lead to corrupted data or a crash.
```
Comment (UnePierre) [inline CppCoreGuidelines.md:867]:
```suggestion
Here we made a small error in `use1` that will lead to corrupted data or a crash.
```

**Review by UnePierre — COMMENTED** [review]
Would you approve without the emphasis commas?

**Pull request merged**